css/style.css

```diff
@@ -1,22 +1,34 @@
 /* global */
 @import url("https://fonts.googleapis.com/css?family=Roboto");
 
+html {
+  height: 100%;
+}
+
 body {
-  /*
-  margin: 0;
-  padding: 0; */
+  display: table;
+  position: absolute;
+  height: 100%;
+  width: 100%;
```

Comment:
meant to fill up the desired area so that the jumbotron class can center-out vertically.